karma.conf.js

```diff
@@ -53,11 +53,11 @@ if (process.env.SAUCE_ACCESS_KEY) {
       browserVersion: "latest",
       "moz:debuggerAddress": true
     },
-    sl_safari_12_1: {
+    sl_safari_16_4: {
       base: "SauceLabs",
       browserName: "safari",
       platform: "macOS 10.13",
-      version: "12.1"
+      version: "16.4"
```

Comment:
@seanpdoyle I got a CI error due to `attachInternals` failing in old Safari. I wondered why [it passed](https://github.com/basecamp/trix/actions/runs/11106544521/job/30855107095) the other day in your PR. It seems it didn't run the sauce tests there?

Reply:
That's surprising. 

The merge commit [failed against `main`](https://github.com/basecamp/trix/actions/runs/11120540387/job/30916158480), but passed (or did not run) [on the PR](https://github.com/basecamp/trix/pull/1188/checks).

Reply:
CI [reads the `SAUCE`-prefixed environment variables from GitHub action secrets](https://github.com/basecamp/trix/blob/c85546ac94735cbf70ffb31b2aef1c327c9e8cb7/.github/workflows/ci.yml#L24-L25), sets them as environment variables, then [reads those values as part of the Karma setup](https://github.com/basecamp/trix/blob/c85546ac94735cbf70ffb31b2aef1c327c9e8cb7/karma.conf.js#L32).

However, 

> With the exception of GITHUB_TOKEN, [secrets are not passed to the runner when a workflow is triggered from a forked repository.](https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions#using-secrets-in-a-workflow)

That means unless there is an alternative solution, pull requests from the community won't run in Sauce Labs managed environments. That seems like a risk to the project's backwards compatibility goals.